packages/celer-web-app/src/data/storage/RecentPages.test.ts

```diff
@@ -0,0 +1,57 @@
+import { addPageToRecents, loadRecentPages, MAX_RECENT_PAGES } from "data/storage";
+// const RecentPages = require("./RecentPages.ts");
+
+// Note that as of Jest@24, localStorage is mocked, so no changes made here
+// should affect the actual values stored.
+const pages = new Array<string>(MAX_RECENT_PAGES + 1);
+for (let i = 0; i < MAX_RECENT_PAGES + 1; i++) {
+	pages[i] = "page" + i;
+}
+test("LocalStorage is initially empty.", () => {
```

Comment:
Put these in a test suite with `describe("data/Storage/RecentPages", () => { ... })`